src/arguments/snowflake.ts

```diff
@@ -18,8 +18,9 @@ export default class extends Argument {
 		if (!arg) throw message.language.get(LanguageKeys.Resolvers.InvalidSnowflake, { name: possible.name });
 
 		if (this.kRegExp.test(arg)) {
-			const snowflake = new Snowflake(arg);
-			if (snowflake.timestamp >= this.kMinimum && snowflake.timestamp < Date.now()) return arg;
+			const snowflake = DiscordSnowflake.deconstruct(arg);
+			const timestamp = Number(snowflake);
```

Comment:
```suggestion
			const timestamp = Number(snowflake.timestamp);
```

Reply:
Resolved all requested changes in 9601852.